apis/logging/v1/elasticsearch_types.go

```diff
@@ -163,10 +183,12 @@ type ElasticsearchNodeSpec struct {
 	// +optional
 	Image string `json:"image,omitempty"`
 
-	// The resource requirements for the Elasticsearch nodes
+	// Limits describes the minimum/maximum amount of compute resources required/allowed
```

Comment:
ResourceRequirements are not only limits, but also requests. AFAIK if you omit the description you will get the default description from `corev1.ResourceRequirements` type, which is a super description anyway. Of course you can copy it here and adapt it to explain things more for elasticsearch, e.g. `the requests/limits for cpu/mem per elasticsearch node`

Reply:
copied from: https://github.com/openshift/elasticsearch-operator/blob/2a23f7cc1b904d9d34e00f7c84f671004eb1930c/manifests/4.7/elasticsearch-operator.v4.7.0.clusterserviceversion.yaml#L341

Reply:
Let's improve this then!